src/app/shared/components/toc/toc.component.scss

```
@@ -8,7 +8,7 @@
   position: fixed;
   top: 90px;
   right: 20px;
-  max-height: 350px;
+  max-height: 600px;
```

```suggestion
    max-height: 100%;
    min-height: 350px;
```

Wouldn't that make more sense? :)